docs/containers/kubernetes-env-vars.md

```diff
@@ -0,0 +1,103 @@
+---
+Area: containers
+ContentId: 8c1ee589-25bb-4c2d-ab8d-7ccfa7b4e282
+PageTitle: Use Kubernetes service environment variables for service to service communication
+DateApproved: 02/12/2021
+MetaDescription: Learn how to use Kubernetes service environment variables with Bridge to Kubernetes to enable service-to-service communication as a non-elevated user in a Kubernetes cluster
+---
+# Kubernetes service environment variables
+
+When you communicate with another service in the same Kubernetes cluster, for example with an HTTP request, you typically use the hardcoded service name in the URL for the request, but that won't work in some scenarios with Bridge to Kubernetes. This article describes how to use the Kubernetes service environment variables to specify the connection Uri.
```

Comment:
Should "Uri" be "URI" in the text? 
Not in the code samples as the type is "Uri"